lib/pry.rb

```diff
@@ -130,6 +130,7 @@ module ExtendCommandBundle; end
 require 'tempfile'
 require 'pathname'
 
+require "pry/rescueable"
```

Comment:
this is spelt `rescueable` but you spell it `rescuable` (without the `e) in other places